niceml/experiments/expfilenames.py

```diff
@@ -74,12 +75,24 @@ class ExpEvalCopyNames:  # pylint: disable=too-few-public-methods
     """Class for determining whether a file should be copied during eval job"""
 
     def __init__(self, exclude_files: Optional[List[str]] = None):
+        """Class for determining whether a file should be copied during eval job"""
+
         self.exclude_files = exclude_files or [
             ExperimentFilenames.ANALYSIS_FOLDER,
             ExperimentFilenames.PREDICTION_FOLDER,
         ]
 
     def __contains__(self, key: str) -> bool:
+        """
+        The __contains__ function is used to check if a key is in the dictionary.
+        This function will be called when using the 'in' keyword.
+
+        Args:
+            key: str: Check if the file name starts with an excluded file
```

Comment:
What does 'file' in this context mean? Where does it come from?
I think what is meant here is 'key' instead of 'file'.

Please rewrite the docstring to better describe what is done here.

Reply:
Renamed to key. file is because we are in the context of the excluded files.